text/machine_comprehension/bert-squad/README.md

```diff
@@ -0,0 +1,111 @@
+# BERT-Squad
+
+## Use cases
+This model uses masked language method to enable training of bidirectional models that can quickly and efficiently run through language processing tasks. The model also adds a next sentence prediction task to improve sentence-level understanding which helps to answer questions based on the given paragraph input. 
+
+## Description
+BERT (Bidirectional Encoder Representations from Transformers) applies Transformers, a popular attention model, to language modelling. This mechanism has an encoder to read the input text and a decoder that produces a prediction for the task. This model uses the technique of masking out some of the words in the input and then condition each word bidirectionally to predict the masked words. BERT also learns to model relationships between sentences, predicts if the sentences are connected or not. 
```

Comment:
Need to list the accuracy of this model on the Squad dataset.

Reply:
I added the accuracy in the validation accuracy section.